remoting/pom.xml

```diff
@@ -45,5 +45,22 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-tcnative-boringssl-static</artifactId>
```

Comment:
`boringssl` is too heavy for `client` module, how about excluding it and use a more lightweight way in `client` module?